website/docs/usage/linguistic-features.md

```diff
@@ -765,15 +765,15 @@ import Tokenization101 from 'usage/101/\_tokenization.md'
 
 <Accordion title="Algorithm details: How spaCy's tokenizer works" id="how-tokenizer-works" spaced>
 
-spaCy introduces a novel tokenization algorithm, that gives a better balance
-between performance, ease of definition, and ease of alignment into the original
+spaCy introduces a novel tokenization algorithm that gives a better balance
+between performance, ease of definition and ease of alignment into the original
 string.
 
 After consuming a prefix or suffix, we consult the special cases again. We want
 the special cases to handle things like "don't" in English, and we want the same
 rule to work for "(don't)!". We do this by splitting off the open bracket, then
-the exclamation, then the close bracket, and finally matching the special case.
-Here's an implementation of the algorithm in Python, optimized for readability
+the exclamation, then the closed bracket, and finally matching the special case.
```

Comment:
Is it "closed bracket" or "closing bracket"?

```suggestion
the exclamation, then the closing bracket, and finally matching the special case.
```

Reply:
I only really know it as "closed bracket". I remember it gave me slight pause. And looking it up, "closed" seemed to be the common way. But it might be changing or different in code?

Reply:
Ah okay, in that case, let's use "closed". It's really just referring to this character here: `)`